### PR TITLE
Make it possible to see janitor logs

### PR DESCRIPTION
**Pull request by Katharine**
This was not set up terribly usefully before.

### CODE REVIEW

File: boskos/janitor/janitor.go [hunk 1]
```diff
@@ -94,7 +101,7 @@ func janitorClean(resource *common.Resource, flags []string) error {
 	cmd := exec.Command(*janitorPath, args...)
 	b, err := cmd.CombinedOutput()
 	if err != nil {
-		logrus.WithError(err).Debugf("failed to clean up project %s, error info: %s", resource.Name, string(b))
+		logrus.WithError(err).Infof("failed to clean up project %s, error info: %s", resource.Name, string(b))
```
Comment (fejta) [inline boskos/janitor/janitor.go]: Would error make more sense here or do we think its too noisy?

**Comment by BenTheElder** [comment]
delayed webhook delivery?

File: boskos/janitor/janitor.go [hunk 2]
```diff
@@ -94,7 +101,7 @@ func janitorClean(resource *common.Resource, flags []string) error {
...
 	} else {
 		logrus.Tracef("output from janitor: %s", string(b))
```
Comment (ixdy) [inline boskos/janitor/janitor.go]: it might be nice to log the output on failures, too?

**Pull request merged**